core/domain/exp_jobs_one_off.py

```diff
@@ -602,13 +602,26 @@ def entity_classes_to_map_over(cls):
     def map(item):
         if item.deleted:
             return
+        err_dict = {}
+        is_error_in_loading = False
```

Comment:
Drop this var. Yield the error and then return. See @prasanna08's PR. https://github.com/oppia/oppia/blob/develop/core/domain/exp_jobs_one_off.py#L353

Don't bundle system errors with errors that are generated from validate_rte_format and that you're already expecting.

In general follow the style in @prasanna08's PR. He's done it correctly.